cdap-ui/app/hydrator/services/create/stores/config-store.js

```diff
@@ -252,9 +254,8 @@ class HydratorPlusPlusConfigStore {
     }
 
     let appType = this.getAppType();
-    if ( this.GLOBALS.etlBatchPipelines.indexOf(appType) !== -1) {
-      config.schedule = this.getSchedule();
-      config.engine = this.getEngine();
+    // Resources
+    if (appType !== this.GLOBALS.etlRealtime) {
```

Comment:
you can remove this check. etlRealtime had already been deprecated, so if this applies to all the 3 types, just set it here without condition

Reply:
Gotcha, sounds good, fixed!